WooCommerce/UITestsFoundation/Screens/Login/LoginEpilogueScreen.swift

```diff
@@ -1,23 +1,33 @@
+import ScreenObject
 import XCTest
 
-private struct ElementStringIDs {
-    static let emailField = "email-label"
-    static let siteUrlField = "url-label"
-    static let continueButton = "login-epilogue-continue-button"
-}
+public final class LoginEpilogueScreen: ScreenObject {
+
+    private let emailLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["email-label"]
+    }
 
-public final class LoginEpilogueScreen: BaseScreen {
-    private let continueButton: XCUIElement
-    private let emailField: XCUIElement
-    private let siteUrlField: XCUIElement
+    private let urlLabelGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["url-label"]
+    }
 
-    public init() {
-        let app = XCUIApplication()
-        emailField = app.staticTexts[ElementStringIDs.emailField]
-        siteUrlField = app.staticTexts[ElementStringIDs.siteUrlField]
-        continueButton = app.buttons[ElementStringIDs.continueButton]
+    private let continueButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["login-epilogue-continue-button"]
+    }
 
-        super.init(element: continueButton)
+    private var actualEmail: String { emailLabelGetter(app).label }
+    private var actualSiteUrl: String { urlLabelGetter(app).label }
```

Comment:
I can see from the history that these two belonged inside asserting function, so the name made sense. Now that they are defined on the higher (lower?🤔) level, I don't think we should use prefixes such as actual/expected, since these elements can be used for different scenarios.

WDYT of renaming them to something neutral (e.g. `emailLabel`...) and either using them in the asserting function below as they are, or to define the actual/expect values inside that function?

Reply:
Ah ok good point, I'll rename them to a more neutral name so that they can be reused.